djangocms_versioning/cms_config.py

```diff
@@ -141,7 +143,19 @@ def label_from_instance(obj, language):
     return "{title} ({path})".format(title=obj.get_title(language), path=obj.get_path(language))
 
 
+def emit_page_change(version):
+    # Trigger a post object operation
+    post_obj_operation.send(
+        sender=version.__class__,
+        operation=CHANGE_PAGE,
```

Comment:
Shouldn't this be CHANGE_PAGE_TRANSLATION?

Reply:
Right, gotcha.